src/components/BrowserCell/BrowserCell.react.js

```diff
@@ -168,7 +168,7 @@ export default class BrowserCell extends Component {
     }
     this.onContextMenu = this.onContextMenu.bind(this);
 
-    if (this.props.markRequiredField && this.props.isRequired && !this.props.value) {
+    if (this.props.markRequiredField && this.props.isRequired && (this.props.value == null || this.props.value === '')) {
```

Comment:
I wonder whether we should allow empty strings (`''`)? An app may have logic that differentiates between `undefined` and empty string. Dashboard currently seems to allow to enter both, e.g. delete all chars in a text field leaves an empty string, deleting the field (select and del key) sets it to undefined. This change seems to take away the possibility to enter an empty string?

Reply:
I think you're right. We should stick to the Server's definition of "valid", and it actually accepts empty strings.